.github/CHANGELOG.md

```diff
@@ -357,42 +400,32 @@
   ```python
   from pennylane.circuit_drawer import MPLDrawer
 
-  drawer = MPLDrawer(n_wires=5, n_layers=5)
+  drawer = MPLDrawer(n_wires=3, n_layers=3)
 
-  drawer.label(["0", "a", r"$|\Psi\rangle$", r"$|\theta\rangle$", "aux"])
+  drawer.label([r"$|\Psi\rangle$", r"$|\theta\rangle$", "aux"])
 
-  drawer.box_gate(layer=0, wires=[0, 1, 2, 3, 4], text="Entangling Layers", text_options={'rotation': 'vertical'})
+  drawer.box_gate(layer=0, wires=[0, 1, 2], text="Entangling Layers", text_options={'rotation': 'vertical'})
   drawer.box_gate(layer=1, wires=[0, 1], text="U(θ)")
 
-  drawer.box_gate(layer=1, wires=4, text="Z")
-
-  drawer.SWAP(layer=1, wires=(2, 3))
-  drawer.CNOT(layer=2, wires=(0, 2))
-
-  drawer.ctrl(layer=3, wires=[1, 3], control_values = [True, False])
-  drawer.box_gate(layer=3, wires=2, text="H", box_options={'zorder': 4},
-    text_options={'zorder': 5})
-
-  drawer.ctrl(layer=4, wires=[1, 2])
-
-  drawer.measure(layer=5, wires=0)
+  drawer.CNOT(layer=2, wires=[1, 2])
+  drawer.measure(layer=3, wires=2)
 
   drawer.fig.suptitle('My Circuit', fontsize='xx-large')
   ```
 
-  <img src="https://pennylane.readthedocs.io/en/latest/_static/drawer/example_basic.png" width=70%/>
+  <img src="https://pennylane.readthedocs.io/en/latest/_static/drawer/example_release_notes.png" width=70%/>
 
 * The slowest tests, more than 1.5 seconds, now have the pytest mark `slow`, and can be
   selected or deselected during local execution of tests.
   [(#1633)](https://github.com/PennyLaneAI/pennylane/pull/1633)
 
-* Hamiltonians are now natively supported on the `default.qubit` device if `shots=None`. 
+* Hamiltonians are now natively supported on the `default.qubit` device if `shots=None`.
   This makes VQE workflows a lot faster in some cases.
```

Comment:
Hey @antalszava, it looks like this still needs to be moved to the "Faster, trainable, Hamiltonian simulations" section, and I can't see the plot? 🤔

Reply:
Good catch, thanks! Updated this